src/CfgEditor/CfgEditorPanel.ts

```diff
@@ -118,8 +120,9 @@ export class CfgEditorPanel implements vscode.CustomTextEditorProvider {
     this.updateWebview(document, webviewPanel);
   };
 
-  // TODO Create separate HTML generator class
-  private _getHtmlForWebview(webview: vscode.Webview) {
+  private async _getHtmlForWebview(webview: vscode.Webview) {
```

Comment:
This function is very good to have. With this, existing static html file would be flexible and extensible. :+1: 👍